dune

```diff
@@ -332,162 +332,5 @@
-  stdcompat__fun.ml.in
-  stdcompat__fun.mli.in
-  stdcompat__fun_s.mli.in
-  stdcompat__hashtbl.ml.in
-  stdcompat__hashtbl.mli.in
-  stdcompat__hashtbl_ext.ml.in
-  stdcompat__hashtbl_ext.mli.in
-  stdcompat__hashtbl_s.mli.in
-  stdcompat__in_channel.ml.in
-  stdcompat__in_channel.mli.in
-  stdcompat__in_channel_s.mli.in
-  stdcompat__init.ml.in
-  stdcompat__init.mli.in
-  stdcompat__int.ml.in
-  stdcompat__int.mli.in
-  stdcompat__int32.ml.in
-  stdcompat__int32.mli.in
-  stdcompat__int32_s.mli.in
-  stdcompat__int64.ml.in
-  stdcompat__int64.mli.in
-  stdcompat__int64_s.mli.in
-  stdcompat__int_s.mli.in
-  stdcompat__lazy.ml.in
-  stdcompat__lazy.mli.in
-  stdcompat__lazy_s.mli.in
-  stdcompat__lexing.ml.in
-  stdcompat__lexing.mli.in
-  stdcompat__lexing_s.mli.in
-  stdcompat__list.ml.in
-  stdcompat__list.mli.in
-  stdcompat__listLabels.ml.in
-  stdcompat__listLabels.mli.in
-  stdcompat__listLabels_s.mli.in
-  stdcompat__list_s.mli.in
-  stdcompat__map.ml.in
-  stdcompat__map.mli.in
-  stdcompat__map_s.mli.in
-  stdcompat__moreLabels.ml.in
-  stdcompat__moreLabels.mli.in
-  stdcompat__moreLabels_s.mli.in
-  stdcompat__native.ml_byte.in
-  stdcompat__native.ml_native.in
-  stdcompat__nativeint.ml.in
-  stdcompat__nativeint.mli.in
-  stdcompat__nativeint_s.mli.in
-  stdcompat__option.ml.in
-  stdcompat__option.mli.in
-  stdcompat__option_s.mli.in
-  stdcompat__out_channel.ml.in
-  stdcompat__out_channel.mli.in
-  stdcompat__out_channel_s.mli.in
-  stdcompat__pervasives.ml.in
-  stdcompat__pervasives.mli.in
-  stdcompat__pervasives_s.mli.in
-  stdcompat__printexc.ml.in
-  stdcompat__printexc.mli.in
-  stdcompat__printexc_s.mli.in
-  stdcompat__printf.ml.in
-  stdcompat__printf.mli.in
-  stdcompat__printf_s.mli.in
-  stdcompat__queue.ml.in
-  stdcompat__queue.mli.in
-  stdcompat__queue_s.mli.in
-  stdcompat__random.ml.in
-  stdcompat__random.mli.in
-  stdcompat__random_s.mli.in
-  stdcompat__result.ml.in
-  stdcompat__result.mli.in
-  stdcompat__result_s.mli.in
-  stdcompat__root.ml.in
-  stdcompat__root.mli.in
-  stdcompat__seq.ml.in
-  stdcompat__seq.mli.in
-  stdcompat__seq_s.mli.in
-  stdcompat__set.ml.in
-  stdcompat__set.mli.in
-  stdcompat__set_s.mli.in
-  stdcompat__stack.ml.in
-  stdcompat__stack.mli.in
-  stdcompat__stack_s.mli.in
-  stdcompat__stdlib.ml.in
-  stdcompat__stdlib.mli.in
-  stdcompat__stdlib_s.mli.in
-  stdcompat__string.ml.in
-  stdcompat__string.mli.in
-  stdcompat__stringLabels.ml.in
-  stdcompat__stringLabels.mli.in
-  stdcompat__stringLabels_s.mli.in
-  stdcompat__string_s.mli.in
-  stdcompat__stubs.c.in
-  stdcompat__sys.ml.in
-  stdcompat__sys.mli.in
-  stdcompat__sys_s.mli.in
-  stdcompat__tools.ml.in
-  stdcompat__tools.mli.in
-  stdcompat__uchar.ml.in
-  stdcompat__uchar.mli.in
-  stdcompat__uchar_s.mli.in
-  stdcompat__unit.ml.in
-  stdcompat__unit.mli.in
-  stdcompat__unit_s.mli.in
-  stdcompat__weak.ml.in
-  stdcompat__weak.mli.in
-  stdcompat__weak_s.mli.in
-  stdcompat_tests.ml.in)
+  (deps (universe))
```

Comment:
```suggestion
  (deps (source_tree .))
```

Reply:
It didn’t work with just universe for me though. It was missing META.in. Maybe we want universe and source_tree

Reply:
really? Wouldn't that be a bug in dune to miss a file? What sort of failure message did you get?

Reply:
The error was:

```
ocaml5.2.0-beta1-stdcompat> File "dune", line 132, characters 0-4956:
ocaml5.2.0-beta1-stdcompat> 132 | (rule
ocaml5.2.0-beta1-stdcompat> 133 |  (targets
ocaml5.2.0-beta1-stdcompat> 134 |   META
ocaml5.2.0-beta1-stdcompat> .....
ocaml5.2.0-beta1-stdcompat> 334 |   stdcompat__weak_s.mli)
ocaml5.2.0-beta1-stdcompat> 335 |   (deps (universe))
ocaml5.2.0-beta1-stdcompat> 336 |   (action (run sh ./configure)))
```

Reply:
It works with both `(universe)` and `(source_tree .)`, so I've readded `(universe)` back.